tests/integration_tests/bugs/test_lp1835584.py

```diff
@@ -0,0 +1,63 @@
+""" Integration test for LP #1835584
+
+Kernels prior to 4.15 uppercased the dmi product_uuid, Azure
+relies on system UUID as their "instance-id". Upgrades to kernels >= 4.15
+will represent the same product_uuid as lowercase. System UUID should be a
+case-insensitive comparison.
+
+The test will launch an Ubuntu PRO FIPS image on Azure which has a 4.15 kernel.
```

Comment:
Does this need to be a Pro image? Could we use an older general-purpose Ubuntu image to bring this test more closely inline with the rest of our testing?

Reply:
In our discussion, the only real path for this currently is Bionic Ubuntu PRO FIPS Azure -> non-FIPS linux-azure kernel. Even linux-azure on Xenial still exhibits a product_uuid that is uppercase and matches the FIPS kernel product_uuid. So, I think we have to limit this test to Bionic to expose the real upgrade path issue folks are seeing.